src/content/docs/iast/install.mdx

```diff
@@ -9,44 +9,179 @@ freshnessValidatedDate: never
 ---
 
 <Callout variant="important">
-  Run IAST with non-production deployments only. IAST tests by invoking HTTP requests with an exploit payload. You must use IAST in non-production environments with only simulated data to avoid both data corruption and introducing exploits into your live code. In addition, when you run IAST on new code in pre-production, you catch potential vulnerabilities before they go live.
+  Run IAST with non-production deployments only. IAST tests the application by
+  invoking HTTP requests with an exploit payload. You must use IAST in
+  non-production environments with only simulated data to avoid both data
+  corruption and introducing exploits into your live code. In addition, when you
+  run IAST on new code in pre-production, you catch potential vulnerabilities
+  before they go live.
 </Callout>
 
+## Install New Relic IAST
+
+There are primarily five stages involved in successful installation of IAST:
+
+1. [Safety check](#safety-check)
+2. [Selection of application language](#selection-of-application-language)
+3. [Updating the APM agent](#updating-the-apm-agent)
+4. [Setting up the security agent](#setting-up-the-security-agent)
+5. [Restart and test the application](#restart-and-test-the-application)
+
+<CONTRIBUTOR_NOTE>
+I may need to delete the above steps as these steps do not align with our Style guide.
+</CONTRIBUTOR_NOTE>
+
 To install New Relic IAST:
 
 <Steps>
   <Step>
-    Go to <DNT>**[one.newrelic.com](https://one.newrelic.com) > All capabilities > IAST**</DNT> and click <DNT>**Set up IAST with applications**</DNT>.
+    Go to <DNT>**[one.newrelic.com](https://one.newrelic.com) > All capabilities > IAST** </DNT> and click <DNT>**Install**</DNT>.  Click <DNT>**Start trial**</DNT> to start a free 30-day trial of IAST.
```

Comment:
There are two steps here.
Follow the <reason>, <location>, <action> format for instructions.